Sources/PrinceOfVersions/Objective-C Helpers/Objective-C Extensions/CheckUpdateFromAppStoreObjectiveCExtensions.swift

```diff
@@ -88,30 +88,42 @@ extension PrinceOfVersions {
 
      If parameter `notificationFrequency` is set to `.always`  and latest version of the app is bigger than installed version, method will always return `.newUpdateAvailable`. However, if the`notificationFrequency` is set to `.once`, only first time this method is called for the same latest app version, it will return `.newUpdateAvailable`, each subsequent call, it will return `.noUpdateAvailable`.
 
+     If the optional `country` parameter is provided, the API request will target the specified App Store region (e.g., `"no"` for Norway). If `country` is not provided, the API will fallback to its default behavior, typically fetching data from the U.S. App Store.
```

Comment:
Let's put a little easter egg in here :) :)

Set country example to hr (Croatia) or mk (Macedonia), you pick :)